jwql/utils/preview_image.py

```diff
@@ -369,7 +371,16 @@ def make_image(self, max_img_size=8):
             diff_img = np.expand_dims(diff_img, axis=0)
         nint, ny, nx = diff_img.shape
 
-        for i in range(nint):
+        # If there are 10 integrations or less, make image for every integration
+        # If there are more than 20 integrations, then make image for every 10th integration
+        if nint <= 10:
+            integration_range = range(nint)
+        elif 11 <= nint <= 100:
+            integration_range = range(0, nint, 10)
+        else:
+            integration_range = range(0, nint, 100)
+
+        for i in integration_range:
```

Comment:
The comment says those with >20 ints will have an image for every 10th int, but it looks like the code is set for this to happen for any with >10 ints. And also any with >100 ints only get an image every 100th int. So just making sure this was the intent.

Reply:
Also, I was hoping to test this, but I was unable to find a preview image with more than 1 intergration. Is this expected at the moment, or was I just unlucky with my random image selections?

Reply:
@bsunnquist Nice catch, I had made changes to how the `integration_range` was determined without updating the inline comments.  I've updated that now. 

I think you may have just had bad luck in finding an image with many integrations.  There are plenty of them in ths new DMS build filesystem (e.g. `jw00733030001_gs-fg_2020102051059` has tens of thousands of integrations 😱 )

The https://github.com/bourque/jwql/tree/bug-fixes-stakeholder-presentations branch that I mentioned above contains fixes for the View Header issues, so those will be solved there.

Thanks again for the very helpful code review!!

Reply:
ah ok, proposal 00733 isn't available for me (I guess because I'm running locally)